doc/chapter-firststeps.asciidoc

```diff
@@ -53,6 +53,11 @@ You can also configure local files as feeds, by prefixing the local path with
 
 	file:///var/log/rss_eventlog.xml
 
+Comments in the _urls_ file start with a `#` and end at the end of the line.
+Newsboat ignores thoses lines. Please note, that commenting out URLs for
```

Comment:
This hints at, but doesn't explicitly state, that comments should be on lines of their own. Maybe something like this?

```suggestion
The _urls_ file can also contain comments: lines that start with `#` and can contain anything you want. Comments are ignored by Newsboat, but can serve as documentation for you.
Please note, that commenting out URLs for
```